### DISCUSSION TITLE
deployment  locally

### DISCUSSION THREAD

**Discussion by kirankumar284** [codinasion-archive/codinasion-monorepo#4584] [General]
I run pnpm install
but I can't run "pnpm run dev" command
![Screenshot from 2023-08-26 10-45-47](https://github.com/codinasion/codinasion/assets/66901994/43a55af0-956f-4c2f-b59d-e735900bc096)

**Comment by harshraj8843** [comment]
Hey @123456-kiran

The path doesn't seem right 🤔

Any command will work only inside the `codinaison` folder

---

Please let us know if the issue persists
